tests/vmemcache_test_mt.c

```diff
@@ -266,6 +267,52 @@ run_test_get_put(VMEMcache *cache, unsigned n_threads, os_thread_t *threads,
 	printf("%s: PASSED\n", __func__);
 }
 
+/*
+ * on_miss_cb -- (internal) 'on miss' callback for run_test_get_on_miss
+ */
+static int
+on_miss_cb(VMEMcache *cache, const void *key, size_t key_size, void *arg)
```

Comment:
you should verify that this is called the expected number of times.

Reply:
It cannot be verified, because sizes of cache entries have random values and therefore a number the 'on miss' callback is called is also random.